### ISSUE TITLE
Lacking strict statements

### ISSUE THREAD

**Issue by kelvinmfs** [dlmanning/gulp-sass#684]
When I try the latest version 4.0.1, it returns the following error.
```
SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
     at exports.runInThisContext (vm.js:53:16)
     at Module._compile (module.js:373:25)
     at Object.Module._extensions..js (module.js:416:10)
     at Module.load (module.js:343:32)
     at Function.Module._load (module.js:300:12)
     at Module.require (module.js:353:17)
     at require (internal/module.js:12:17)
     at Object.<anonymous> (C:\BuildAgent\work\756e6ea965fc640e\source\Dashboard\gulpfile.js:19:16)
     at Module._compile (module.js:409:26)
     at Object.Module._extensions..js (module.js:416:10)
```
It looks like some of your methods lack `"use strict";`

**Comment by xzyfer** [comment]
See https://github.com/dlmanning/gulp-sass/pull/683

**Comment by D3MaxT** [comment]
@xzyfer 
Good to see the fix was commited. Are you going to release it as 4.0.2?

Thank you!

**Comment by xzyfer** [comment]
This will not be fixed. Gulp-sass 4 requires Node 6 or greater. If you need Node 4 you can continue to use gulp-sass 3

**Comment by D3MaxT** [comment]
I am confused. I am running Node 8.11.1 with NPM 5.8.0 (so latest LTS and latest) and this broke Integration with VS2017.

**Comment by xzyfer** [comment]
That commit was not merged. This is not a bug.

This error happens in Node < 6 due to missing support for some es6 features. As if gulp-sass 4 we only support Node 6+.

**Comment by xzyfer** [comment]
As for vs2017, it has its own version of node and doesn't use the system version.

You'll need to check you vs settings to use the correct version of node.

**Comment by D3MaxT** [comment]
@xzyfer 
Maybe I am not communicating this correctly. Or maybe I am just misunderstanding something.

I am not using node that ships with VS2017. I am using the latest version of Node LTS (8.11.1) with the latest NPM.

I added gulp 3.9.1 and gulp-sass 4.0.1 to package.json as dev dependencies.
I configured a simple task in the gulpfile.js, and when I try to run the task (using VS task runner), the error from the original post comes up. The fix was to manually go in and add the 'use strict'; line to the index.js file, and everything works. This is the same fix as in the commit that hasn't been merged yet.

Not sure why it wouldn't be merged into a 4.0.2 release. Using the latest version of node LTS does not work with 4.0.1 without the 'use strict'; fix.

Thank you.

**Comment by FreedCapybara** [comment]
To save somebody some time in figuring out how to fix VS, go to
`Tools > Options > Projects and Solutions > Web Package Management`

There's a version of node in `$(VSINSTALLDIR)\Web\External`, so I bumped up `$(PATH)` using the buttons in the top-right corner to override it. It worked 👍

Here's my final configuration:
![image](https://user-images.githubusercontent.com/7003723/42776772-0e24cc64-8906-11e8-9511-f499ee72d329.png)
